### ISSUE TITLE
not able to navigate to homepage after selecting a series

### ISSUE THREAD

**Issue by krshashwat** [developerrahulofficial/kdrama-streaming#1]
When we select a show then the URL too doesn't change moreover we can't navigate back to homepage as there is no links unless a page refresh is done
![image](https://github.com/developerrahulofficial/kdrama-streaming/assets/25583433/9a272b4c-e7cc-4ea1-b9ff-bb216c685246)

**Comment by developerrahulofficial** [comment]
Yes there is some issues , this is the first version i will update it in
later versions
